uber/models/attendee.py

```diff
@@ -99,6 +99,7 @@ class Attendee(MagModel, TakesPaymentMixin):
 
     # attendee shirt size for both swag and staff shirts
     shirt = Column(Choice(c.SHIRT_OPTS), default=c.NO_SHIRT)
+    second_shirt = Column(Choice(c.SECOND_SHIRT_OPTS), default=c.UNKNOWN)
```

Comment:
In theory the code supports N staff shirts rather than just 2, but I think that ``second_shirt`` is still a good name for this property, since I'd expect our policy to always be that at most 1 shirt would be an event shirt and the the rest would be staff shirts, and this is indeed how this is implemented.